frmts/raw/rrasterdataset.cpp

```diff
@@ -569,6 +569,13 @@ void RRASTERDataset::RewriteHeader()
             VSIFPrintfL(fp, "projection=%s\n", pszProj4);
             VSIFree(pszProj4);
         }
+        char* pszWKT = nullptr;
+        oSRS.exportToWkt(&pszWKT);
```

Comment:
note that currently the default is still WKT1. If you want WKT2 :
```suggestion
        const char* const apszOptions[] = { "FORMAT=WKT2_2019", nullptr };
        oSRS.exportToWkt(&pszWKT, apszOptions);
```